### PR TITLE
[ADD] website: address autocomplete configuration

### PR DESCRIPTION
**Pull request by masp-odoo**
Task-3618264

### CODE REVIEW

File: content/applications/websites/website/configuration/address_autocomplete.rst [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+.. note::
+   For security purposes, you can restrict `the usage of your API key.
+   Under the :guilabel:`"API restrictions"` section, you can specify which APIs your key can access.
+   For the Places API, you can restrict it to only allow requests from specific websites or apps.
+
+.. important::
+   Save Your API Key: Copy your API key and securely store it.
+   Do not share it publicly or expose it in client-side code.
+
+Step 3: Usage
```
Comment (Donapi) [inline content/applications/websites/website/configuration/address_autocomplete.rst:55]: I wouldn't put the info related to the "usage" in a Step 3. 
I would make a new section like "Using Google Places API"

File: content/applications/websites/website/configuration/address_autocomplete.rst [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+-------------
+
+Now that you have your API key, you can start using it in your applications to make requests to the Google Places API.
+Include the API key in your requests using the key parameter.
+
+.. important::
```
Comment (Donapi) [inline content/applications/websites/website/configuration/address_autocomplete.rst:61]: I'm not sure it makes sens to mention this info in Odoo documentation. We can discuss it :)

**Review by Donapi — CHANGES_REQUESTED** [review]
Hi Marion,
Congrats on your first PR; it's good work!
I made some comments but it would be best to discuss it IRL. I was not able to reproduce all the steps you described. Can we check together? And I will also explain some of my comments :)

**Review by jcs-odoo — APPROVED** [review]
@robodoo delegate+

**Comment by masp-odoo** [comment]
@robodoo r+

**Comment by masp-odoo** [comment]
@robodoo r+

**Pull request closed without merging**